rotkehlchen/globaldb/schema.py

```diff
@@ -108,8 +108,11 @@
 INSERT OR IGNORE INTO token_kinds(token_kind, seq) VALUES ('C', 3);
 """
 
-# Using asset_id as a primary key here since nothing else is guaranteed to be unique
-# Advantage we have here is that by deleting the parent asset this also gets deleted
+# The common_assets_details contains information common for all the crypto-assets
+# leaving to the assets table information that is shared among the custom assets.
+# For every assets row we have a row in the common_assets_details table or in the
+# custom_assets table. The identifier of the asset is primary key since this allows as to do
+# a delete on cascade when deleting the row in the asset table.
```

Comment:
If we will keep foreign keys mention then:
```suggestion
# a cascade on delete when deleting the row in the asset table.
```

(same suggestion for other tables)